app/scripts/background.js

```diff
@@ -142,6 +143,24 @@ const PHISHING_WARNING_PAGE_TIMEOUT = ONE_SECOND_IN_MILLISECONDS;
 // Event emitter for state persistence
 export const statePersistenceEvents = new EventEmitter();
 
+if (isFirefox) {
+  browser.runtime.onInstalled.addListener(function (details) {
+    if (details.reason === 'install') {
+      browser.storage.session.set({ isFirstTimeInstall: true });
+    } else if (details.reason === 'update') {
```

Comment:
do we need to handle `browser_update`, `chrome_update`, and `shared_module_update` events the same way as well? https://developer.chrome.com/docs/extensions/reference/api/runtime#type-OnInstalledReason

Reply:
yes, good catch

I think I can just change this to an `else`

Reply:
Switched to using session storage in 45dda9f31d